spectrum/autism_prevalence_map/static/autism_prevalence_map/ts/joint.ts

```diff
@@ -2,6 +2,9 @@ import { app } from './app.js';
 
 export function ttInitJoint() {
     $(document).ready(function () {
+        if (typeof country === 'object') country = '';
+        if (typeof continent === 'object') continent = '';
```

Comment:
these two lines seem like a hack to me. I did notice when switching from map to list, those two parameters appear as object in the query string. Can you dig a bit to find the cause of it?